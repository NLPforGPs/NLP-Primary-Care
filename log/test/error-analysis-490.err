cpu-bind=MASK - bp1-gpu016, task 4294967295 4294967295 [0]: mask 0x2020 set

CondaValueError: prefix already exists: /user/home/zu20361/.conda/envs/NLP_GP

[nltk_data] Downloading package wordnet to
[nltk_data]     /user/home/zu20361/nltk_data...
[nltk_data]   Package wordnet is already up-to-date!
[nltk_data] Downloading package punkt to
[nltk_data]     /user/home/zu20361/nltk_data...
[nltk_data]   Package punkt is already up-to-date!
Some weights of the model checkpoint at microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract-fulltext were not used when initializing BertForSequenceClassification: ['cls.seq_relationship.bias', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.transform.dense.weight', 'cls.predictions.bias', 'cls.predictions.transform.dense.bias', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.decoder.bias', 'cls.seq_relationship.weight', 'cls.predictions.decoder.weight']
- This IS expected if you are initializing BertForSequenceClassification from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertForSequenceClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
Some weights of BertForSequenceClassification were not initialized from the model checkpoint at microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract-fulltext and are newly initialized: ['classifier.weight', 'classifier.bias']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
2021-12-10 14:57:39,908 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:154] - INFO: Predicting...
2021-12-10 14:57:39,988 - /user/home/zu20361/.conda/envs/NLP_GP/lib/python3.9/site-packages/datasets/builder.py[line:377] - WARNING: Using custom data configuration default-a2d0abb6534a5099
0 examples [00:00, ? examples/s]2021-12-10 14:57:40,219 - /user/home/zu20361/.cache/huggingface/modules/datasets_modules/datasets/transcript_evaldataset/9a9f33ed1951859e6f0a536ab67cad76ef5b13196ce8d7b688334d25b0f51aa6/transcript_evaldataset.py[line:60] - INFO: Generating examples...
                                  0%|          | 0/1 [00:00<?, ?it/s]100%|██████████| 1/1 [00:00<00:00, 515.90it/s]
2021-12-10 14:57:40,293 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:201] - INFO: conventional classifier...
Some weights of the model checkpoint at microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract-fulltext were not used when initializing BertForSequenceClassification: ['cls.seq_relationship.bias', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.transform.dense.weight', 'cls.predictions.bias', 'cls.predictions.transform.dense.bias', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.decoder.bias', 'cls.seq_relationship.weight', 'cls.predictions.decoder.weight']
- This IS expected if you are initializing BertForSequenceClassification from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing BertForSequenceClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
Some weights of BertForSequenceClassification were not initialized from the model checkpoint at microsoft/BiomedNLP-PubMedBERT-base-uncased-abstract-fulltext and are newly initialized: ['classifier.weight', 'classifier.bias']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
2021-12-10 14:57:42,745 - /user/home/zu20361/NLP-Primary-Care/./run_plms.py[line:213] - INFO: id to label is {0: 'Superficial vein thrombosis (superficial thrombophlebitis)', 1: 'Bipolar disorder', 2: 'Carpal tunnel syndrome', 3: 'Cholesteatoma', 4: 'DVT prevention for travellers', 5: 'Mumps', 6: 'Acute childhood limp', 7: 'Fungal skin infection - body and groin', 8: 'Depression in children', 9: 'Cervical cancer and HPV', 10: 'Acne vulgaris', 11: 'Obstructive sleep apnoea syndrome', 12: 'Cough', 13: 'Diarrhoea - antibiotic associated', 14: 'Adverse drug reactions', 15: 'Heart failure - chronic', 16: 'Tinnitus', 17: 'Ankylosing spondylitis', 18: 'Febrile seizure', 19: 'Eating disorders', 20: 'Chest pain', 21: 'Whooping cough', 22: 'Psoriasis', 23: 'Otitis externa', 24: 'Corneal superficial injury', 25: 'Gout', 26: 'Fungal skin infection - scalp', 27: 'Croup', 28: 'Vertigo', 29: 'Antenatal care - uncomplicated pregnancy', 30: 'Miscarriage', 31: 'Alcohol - problem drinking', 32: 'Chalazion (meibomian cyst)', 33: 'Palliative care - constipation', 34: 'Pilonidal sinus disease', 35: 'Ovarian cancer', 36: 'Conjunctivitis - infective', 37: 'Headache - medication overuse', 38: 'Candida - oral', 39: 'Plantar fasciitis', 40: 'Urticaria', 41: 'Hepatitis B', 42: "Addison's disease", 43: 'Veruccae and warts', 44: 'Vestibular neuronitis', 45: 'Knee pain - assessment', 46: "Parkinson's disease", 47: 'Pre-conception - advice and management', 48: 'Blepharitis', 49: 'Anaemia - iron deficiency', 50: 'Leg ulcer - venous', 51: 'Bruising', 52: 'Achilles tendinopathy', 53: 'Obesity', 54: 'Bronchiectasis', 55: 'Hypertension', 56: 'Tennis elbow', 57: 'Varicocele', 58: 'Migraine', 59: 'Hypercholesterolaemia - familial', 60: 'Syphilis', 61: 'Neck pain - non-specific', 62: 'Nappy rash', 63: 'Fungal skin infection - foot', 64: 'Headache - assessment', 65: 'Osteoarthritis', 66: 'Fungal nail infection', 67: 'Feverish children - management', 68: 'Gynaecological cancers - recognition and referral', 69: 'Menopause', 70: "Baker's cyst", 71: 'Prostatitis - acute', 72: 'Pulmonary embolism', 73: 'Cardiac arrest - out of hospital care', 74: 'Irritable bowel syndrome', 75: 'Herpes simplex - ocular', 76: 'Polymyalgia rheumatica', 77: 'Aphthous ulcer', 78: 'Vitamin D deficiency in adults', 79: 'Balanitis', 80: 'Chickenpox', 81: 'Food allergy', 82: 'Burns and scalds', 83: 'Faltering growth', 84: 'Sickle cell disease', 85: 'Generalized anxiety disorder', 86: 'Multiple sclerosis', 87: 'Hyperthyroidism', 88: 'Palpitations', 89: 'Mental health in students', 90: "Cow's milk allergy in children", 91: 'Post-traumatic stress disorder', 92: 'Benzodiazepine and z-drug withdrawal', 93: 'Malaria prophylaxis', 94: 'Benign paroxysmal positional vertigo', 95: 'Bites and stings - insect', 96: 'Undescended testes', 97: 'Platelets - abnormal counts and cancer', 98: 'Prostatitis - chronic', 99: 'Hypertension in pregnancy', 100: 'Bursitis - pre-patellar', 101: 'Coronavirus - COVID 19', 102: 'Coeliac disease', 103: 'Back pain - low (without radiculopathy)', 104: 'Dyspepsia - proven GORD', 105: 'Cerebral palsy', 106: 'Sinusitis', 107: 'Autism in children', 108: 'Erythrocytosis/polycythaemia', 109: 'Otitis media with effusion', 110: 'Uveitis', 111: 'Dysmenorrhoea', 112: 'Sore throat - acute', 113: 'Hiccups', 114: 'Depression', 115: 'Palliative care - general issues', 116: 'Dental abscess', 117: 'Warts - anogenital', 118: 'Tiredness/fatigue in adults', 119: 'Bacterial meningitis and meningococcal disease', 120: 'Breast abscess and mastitis', 121: 'Feverish children - risk assessment', 122: "Bell's palsy", 123: 'Contraception - sterilization', 124: 'Colic - renal or ureteric (acute)', 125: 'Diabetes - type 2', 126: 'Urinary tract infection (lower) - men', 127: 'Breast pain - cyclical', 128: 'Menorrhagia', 129: 'Influenza - seasonal', 130: 'Vitiligo', 131: 'Palliative cancer care - pain', 132: 'Carbon monoxide poisoning', 133: 'Ectopic pregnancy', 134: 'Colic - infantile', 135: 'Hordeola (styes)', 136: 'Chilblains', 137: 'Dry eye syndrome', 138: 'Neck pain - whiplash injury', 139: 'Gastrointestinal tract (lower) cancers - recognition and referral', 140: 'Lipodermatosclerosis and venous eczema', 141: 'Scrotal pain and swelling', 142: 'Cellulitis - acute', 143: 'Hyperhidrosis', 144: 'Lung and pleural cancers - recognition and referral', 145: 'Tamoxifen - managing adverse effects', 146: 'Immunizations - pneumococcal', 147: 'Osteoporosis - prevention of fragility fractures', 148: 'Pubic lice', 149: 'Tuberculosis', 150: 'Lyme disease', 151: 'Dyspepsia - proven peptic ulcer', 152: 'Jaundice in adults', 153: 'Shoulder pain', 154: 'Angio-oedema and anaphylaxis', 155: 'Scarlet fever', 156: 'Obsessive-compulsive disorder', 157: 'Common cold', 158: 'Non-alcoholic fatty liver disease (NAFLD)', 159: 'Anaemia - B12 and folate deficiency', 160: 'Teething', 161: 'Earwax', 162: 'Urinary tract infection (lower) - women', 163: 'Herpes simplex - genital', 164: 'Prostate cancer', 165: "Morton's neuroma", 166: 'Candida - female genital', 167: "Meniere's disease", 168: 'Leg cramps', 169: 'Acute kidney injury', 170: 'Constipation in children', 171: 'Roundworm', 172: 'Sprains and strains', 173: 'Breastfeeding problems', 174: 'Vitamin D deficiency in children', 175: 'Headache - cluster', 176: "Crohn's disease", 177: 'Antenatal and postnatal depression', 178: 'Diarrhoea - prevention and advice for travellers', 179: 'Osgood-Schlatter disease', 180: 'Infertility', 181: 'MI - secondary prevention', 182: "Gilbert's syndrome", 183: 'Breast cancer - managing FH', 184: 'Cholecystitis - acute', 185: 'Nausea/vomiting in pregnancy', 186: 'Glaucoma', 187: 'Neuropathic pain - drug treatment', 188: 'Opioid dependence', 189: 'Epistaxis (nosebleeds)', 190: 'Diabetes - type 1', 191: 'Haemorrhoids', 192: 'Headache - tension-type', 193: 'Asthma', 194: 'Haematological cancers - recognition and referral', 195: 'Herpetic whitlow - and staphylococcal', 196: 'Otitis media - chronic suppurative', 197: 'Radiculopathy (lumbar) - sciatica', 198: 'Trigeminal neuralgia', 199: 'Squint in children', 200: 'Angina', 201: "Dupuytren's disease", 202: 'Appendicitis', 203: 'Otitis media - acute', 204: 'Palliative care - nausea and vomiting', 205: 'Pruritus ani', 206: 'Eczema - atopic', 207: 'Cataracts', 208: 'Scabies', 209: 'Hepatitis C', 210: 'Hearing loss in adults', 211: 'Olecranon bursitis', 212: 'Peripheral arterial disease', 213: 'Self-harm', 214: 'Bedwetting (enuresis)', 215: 'Sepsis', 216: "Raynaud's phenomenon", 217: 'Multiple myeloma', 218: 'Restless legs syndrome', 219: 'Skin cancers - recognition and referral', 220: 'GORD in children', 221: 'Pruritus vulvae', 222: 'Chest infections - adult', 223: 'Poisoning or overdose', 224: 'Cough - acute with chest signs in children', 225: 'Chronic obstructive pulmonary disease', 226: 'End of life care in children', 227: 'Hyponatraemia', 228: 'Fibroids', 229: 'Erectile dysfunction', 230: 'Chronic kidney disease', 231: 'Gingivitis and periodontitis', 232: 'Measles', 233: 'Compression stockings', 234: 'Shingles', 235: 'Hand foot and mouth disease', 236: 'Rubella', 237: 'Ulcerative colitis', 238: 'Temporomandibular disorders (TMDs)', 239: 'Blackouts', 240: 'Insomnia', 241: 'Alopecia areata', 242: 'Molluscum contagiosum', 243: 'Allergic rhinitis', 244: 'Gonorrhoea', 245: 'Delirium', 246: 'Malaria', 247: 'Paronychia - acute', 248: 'Sleep disorders - shift work and jet lag', 249: 'Hirsutism', 250: 'Gallstones', 251: 'Incontinence - urinary, in women', 252: 'Parvovirus B19 infection', 253: 'Attention deficit hyperactivity disorder', 254: 'Pityriasis versicolor', 255: 'Itch - widespread', 256: 'Haematospermia', 257: 'Chlamydia - uncomplicated genital', 258: 'MRSA in primary care', 259: 'Anal fissure', 260: 'Neutropenic sepsis', 261: 'Pityriasis rosea', 262: 'Hepatitis A', 263: 'Age-related macular degeneration', 264: 'Diverticular disease', 265: 'Pyelonephritis - acute', 266: 'Seborrhoeic dermatitis', 267: 'Hypercalcaemia', 268: 'Psychosis and schizophrenia', 269: 'Amenorrhoea', 270: 'Atrial fibrillation', 271: 'Dementia', 272: 'Dyspepsia - unidentified cause', 273: 'Deep vein thrombosis', 274: 'Threadworm', 275: 'Dyspepsia - proven functional', 276: 'Gastroenteritis', 277: 'Bunions', 278: 'Alopecia, androgenetic - male', 279: 'Rosacea', 280: 'Chronic pain', 281: 'Cervical radiculopathy - neck pain', 282: 'Palliative care - oral', 283: 'Constipation', 284: 'Epilepsy', 285: 'Neck pain - acute torticollis', 286: 'Jaundice in the newborn', 287: 'Neck lump', 288: 'Polycystic ovary syndrome', 289: 'Premenstrual syndrome', 290: 'Dyspepsia - pregnancy-associated', 291: 'Greater trochanteric pain syndrome', 292: 'Learning disabilities', 293: 'Dermatitis - contact', 294: 'Breathlessness', 295: 'Hypothyroidism', 296: 'Gastrointestinal tract (upper) cancers - recognition and referral', 297: 'Cirrhosis', 298: 'Urological cancers - recognition and referral', 299: 'Alopecia, androgenetic - female', 300: "Diarrhoea - adult's assessment", 301: 'Impetigo', 302: 'Neuralgia - post-herpetic', 303: 'Boils, carbuncles, and staphylococcal carriage', 304: 'Halitosis', 305: 'Red eye', 306: 'Retinal detachment', 307: 'Autism in adults', 308: 'Candida - skin', 309: 'Urinary tract infection - children', 310: 'Conjunctivitis - allergic', 311: 'Urethritis - male', 312: 'Herpes simplex - oral', 313: 'Stroke and TIA', 314: 'Vaginal discharge'}
  0%|          | 0/1 [00:00<?, ?ba/s]100%|██████████| 1/1 [00:02<00:00,  2.02s/ba]100%|██████████| 1/1 [00:02<00:00,  2.02s/ba]
  0%|          | 0/2 [00:00<?, ?ba/s] 50%|█████     | 1/2 [00:00<00:00,  1.13ba/s]100%|██████████| 2/2 [00:01<00:00,  1.77ba/s]100%|██████████| 2/2 [00:01<00:00,  1.63ba/s]
  0%|          | 0/43 [00:00<?, ?it/s]  2%|▏         | 1/43 [00:00<00:26,  1.56it/s]  5%|▍         | 2/43 [00:01<00:20,  2.03it/s]  7%|▋         | 3/43 [00:01<00:17,  2.26it/s]  9%|▉         | 4/43 [00:01<00:16,  2.38it/s] 12%|█▏        | 5/43 [00:02<00:15,  2.45it/s] 14%|█▍        | 6/43 [00:02<00:14,  2.50it/s] 16%|█▋        | 7/43 [00:02<00:14,  2.52it/s] 19%|█▊        | 8/43 [00:03<00:13,  2.54it/s] 21%|██        | 9/43 [00:03<00:13,  2.56it/s] 23%|██▎       | 10/43 [00:04<00:12,  2.57it/s] 26%|██▌       | 11/43 [00:04<00:12,  2.57it/s] 28%|██▊       | 12/43 [00:04<00:12,  2.58it/s] 30%|███       | 13/43 [00:05<00:11,  2.58it/s] 33%|███▎      | 14/43 [00:05<00:11,  2.58it/s] 35%|███▍      | 15/43 [00:06<00:10,  2.58it/s] 37%|███▋      | 16/43 [00:06<00:10,  2.58it/s] 40%|███▉      | 17/43 [00:06<00:10,  2.58it/s] 42%|████▏     | 18/43 [00:07<00:09,  2.58it/s] 44%|████▍     | 19/43 [00:07<00:09,  2.58it/s] 47%|████▋     | 20/43 [00:07<00:08,  2.58it/s] 49%|████▉     | 21/43 [00:08<00:08,  2.57it/s] 51%|█████     | 22/43 [00:08<00:08,  2.57it/s] 53%|█████▎    | 23/43 [00:09<00:07,  2.57it/s] 56%|█████▌    | 24/43 [00:09<00:07,  2.57it/s] 58%|█████▊    | 25/43 [00:09<00:07,  2.57it/s] 60%|██████    | 26/43 [00:10<00:06,  2.57it/s] 63%|██████▎   | 27/43 [00:10<00:06,  2.57it/s] 65%|██████▌   | 28/43 [00:11<00:05,  2.57it/s] 67%|██████▋   | 29/43 [00:11<00:05,  2.57it/s] 70%|██████▉   | 30/43 [00:11<00:05,  2.57it/s] 72%|███████▏  | 31/43 [00:12<00:04,  2.57it/s] 74%|███████▍  | 32/43 [00:12<00:04,  2.57it/s] 77%|███████▋  | 33/43 [00:13<00:03,  2.56it/s] 79%|███████▉  | 34/43 [00:13<00:03,  2.56it/s] 81%|████████▏ | 35/43 [00:13<00:03,  2.55it/s] 84%|████████▎ | 36/43 [00:14<00:02,  2.56it/s] 86%|████████▌ | 37/43 [00:14<00:02,  2.56it/s] 88%|████████▊ | 38/43 [00:15<00:01,  2.56it/s] 91%|█████████ | 39/43 [00:15<00:01,  2.54it/s] 93%|█████████▎| 40/43 [00:15<00:01,  2.54it/s] 95%|█████████▌| 41/43 [00:16<00:00,  2.55it/s] 98%|█████████▊| 42/43 [00:16<00:00,  2.55it/s]100%|██████████| 43/43 [00:16<00:00,  3.27it/s]100%|██████████| 43/43 [00:16<00:00,  2.58it/s]

real	0m51.431s
user	0m23.087s
sys	0m9.893s
