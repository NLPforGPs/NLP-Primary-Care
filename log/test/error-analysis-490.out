JOB ID: 275513
Working Directory: /user/home/zu20361/NLP-Primary-Care
Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
Start Time: Fri Dec 10 14:57:17 GMT 2021
loading codes from PE...
Downloading and preparing dataset description_dataset/default (download: Unknown size, generated: 2.07 MiB, post-processed: Unknown size, total: 2.07 MiB) to /user/home/zu20361/.cache/huggingface/datasets/description_dataset/default-a2d0abb6534a5099/0.0.0/9a9f33ed1951859e6f0a536ab67cad76ef5b13196ce8d7b688334d25b0f51aa6...
Dataset description_dataset downloaded and prepared to /user/home/zu20361/.cache/huggingface/datasets/description_dataset/default-a2d0abb6534a5099/0.0.0/9a9f33ed1951859e6f0a536ab67cad76ef5b13196ce8d7b688334d25b0f51aa6. Subsequent calls will reuse this data.
classification_report:
                 precision    recall  f1-score   support

          blood       0.33      0.88      0.48         8
 cardiovascular       0.36      0.88      0.51        32
      digestive       0.53      0.64      0.58        44
            ear       0.82      0.82      0.82        11
      endocrine       0.27      0.62      0.38        24
            eye       0.19      0.60      0.29         5
         female       0.24      0.79      0.37        14
        general       0.06      0.21      0.09        14
           male       0.24      0.71      0.36         7
musculoskeletal       0.63      0.85      0.72        65
   neurological       0.15      0.50      0.23        20
      pregnancy       0.12      0.45      0.19        11
  psychological       0.38      0.82      0.52        50
    respiratory       0.65      0.81      0.72        37
           skin       0.42      0.78      0.55        32
     urological       0.55      0.33      0.41        18

      micro avg       0.36      0.72      0.48       392
      macro avg       0.37      0.67      0.45       392
   weighted avg       0.44      0.72      0.52       392
    samples avg       0.40      0.73      0.49       392

f1_score 0.45150508447067483
End Time: Fri Dec 10 14:58:09 GMT 2021
